File: src/main/java/io/github/takenoha/towerdefense/persistence/TowerUpgradeReceiptState.java
package io.github.takenoha.towerdefense.persistence;

/** Durable handoff state for legacy tower-upgrade materials. */
public enum TowerUpgradeReceiptState {
    RESERVED,
    SECURED,
    RETURN_PENDING,
    CLEAR_PENDING,
    CLEARED,
    RESTORED
}

File: src/main/kotlin/io/github/takenoha/towerdefense/persistence/TowerUpgradeReceiptState.kt
package io.github.takenoha.towerdefense.persistence

/** Durable handoff state for legacy tower-upgrade materials. */
enum class TowerUpgradeReceiptState {
    RESERVED,
    SECURED,
    RETURN_PENDING,
    CLEAR_PENDING,
    CLEARED,
    RESTORED,
}
